### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brianemwd**
Added instructions for setting up site_owner email in mailman-extra.cfg.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -418,6 +418,15 @@ configuration: /etc/postfix-mailman.cfg
 
 The configuration file `/etc/postfix-mailman.cfg` is generated automatically.
 
+# Setup site owner address. By default, mailman is setup with the site_owner set to 'changeme@example.com'. This should be pointing to a valid mailbox. Add the following to the '/opt/mailman/core/mailman-extra.cfg'.
```
Comment (maxking) [inline README.md]: Can you wrap the changes in triple backticks?

```
Something like this
```
So it looks like a "code". By default, in Markdown, `#` is a H1 and not comment.

Comment (brianemwd) [reply README.md]: Ok. I believe I have right now.

Comment (maxking) [reply README.md]: Looks great! Thanks!

**Comment by maxking** [comment]
Thanks for your contribution!

**Pull request merged**